runtime/onert/core/include/util/OnertException.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_UTIL_ONERTEXCEPTION_H__
+#define __ONERT_UTIL_ONERTEXCEPTION_H__
+
+#include <iostream>
+
+namespace onert
+{
+namespace util
+{
+namespace exception
+{
+
+class OnertException : public std::exception
+{
+  std::string message;
+
+public:
+  OnertException(std::string _m) : message(_m) {}
+
+  virtual const char *what() const throw() { return message.c_str(); }
+};
+
+class ExecutionFailedException : public OnertException
+{
+public:
+  ExecutionFailedException(std::string _m) : OnertException("ExecutionFailedException : " + _m) {}
+};
+
+class FileNotFoundException : public OnertException
+{
+public:
+  FileNotFoundException(std::string _m) : OnertException("FileNotFoundException : " + _m) {}
+};
+
+class NotSupportedTypeException : public OnertException
+{
+public:
+  NotSupportedTypeException(std::string _m) : OnertException("NotSupportedTypeException : " + _m) {}
+};
+
+class InvalidValueException : public OnertException
+{
+public:
+  InvalidValueException(std::string _m) : OnertException("InvalidValueException : " + _m) {}
+};
+
+class NotImplementedException : public OnertException
+{
+public:
+  NotImplementedException(std::string _m) : OnertException("NotImplementedException : " + _m) {}
```

Comment:
I think... It is good to simplify the classification of exceptions to the minimum.

For this reason, for example,
Whether or not there is a support plan, it is the same that has not been implemented.
Additional classify there can be confusing for user trying to debug.

I think you can use '//' comments if needed.

Reply:
In addition, I think that even in the case of NotSupportedTypeException, it can be used equally whether the datatype is not supported or the shape is not supported.

Reply:
I classify NotImplementedException -> NotYetImplementedException and NotYetSupportedException

Reply:
I am not sure the difference between `NotYetImplementedException` and `NotYetSupportedException`.

Reply:
Add NotSupportedOperationException and NotYetSupportedOperationException 

NotImplementedException and NotYetImplementedException can be used when there is a part that is supported but not yet implemented. tell me your opinions.